Makefile

```diff
@@ -187,20 +187,14 @@ docker-clean:
 	@echo "Deleting images..."
 	@docker image prune -a -f $(ST_DOCKER_FILTER)
 
-smoke-test: system-test-deps docker smoke-test-run
+smoke-test: docker smoke-test-run
 
 smoke-test-run:
 	@./rebar3 as system_test do ct $(ST_CT_FLAGS) --suite=aest_sync_SUITE,aest_commands_SUITE,aest_peers_SUITE
```

Comment:
The aest_channels suite is probably more a smoke test for API change than aest_peers is. The latter does not use the API, since sync is done via noise.

Reply:
> The aest_channels suite is probably more a smoke test for API change than aest_peers is. The latter does not use the API, since sync is done via noise.

Tracked aest_channels addition in https://github.com/aeternity/epoch/pull/1498